passes/opt/Makefile.inc

```diff
@@ -6,6 +6,7 @@ OBJS += passes/opt/opt_reduce.o
 OBJS += passes/opt/opt_rmdff.o
 OBJS += passes/opt/opt_clean.o
 OBJS += passes/opt/opt_expr.o
+OBJS += passes/opt/opt_demorgan.o
```

Comment:
Should be moved down to the non-small section. Same thing for `rmports` actually.